ui/app/models/azure/config.js

```diff
@@ -65,4 +65,41 @@ export default class AzureConfig extends Model {
   get formFields() {
     return expandAttributeMeta(this, this.configurableParams);
   }
+
+  // "filedGroupsWif" and "fieldGroupsAzure" are passed to the FormFieldGroups component to determine which group to show in the form (ex: @groupName="fieldGroupsWif")
+  get fieldGroupsWif() {
+    return fieldToAttrs(this, this.formFieldGroups('wif'));
+  }
+
+  get fieldGroupsAzure() {
+    return fieldToAttrs(this, this.formFieldGroups('azure'));
+  }
+
+  get isWifPluginConfigured() {
+    return !!this.identityTokenAudience || !!this.identityTokenTtl;
+  }
+
+  get isAzureAccountConfigured() {
+    // clientSecret is not checked here because it's never return by the API
+    // however it is an Azure account field
+    return !!this.rootPasswordTtl;
+  }
+
+  formFieldGroups(accessType = 'azure') {
+    const formFieldGroups = [];
+    formFieldGroups.push({
+      default: ['subscriptionId', 'tenantId', 'clientId', 'environment'],
+    });
+    if (accessType === 'wif') {
+      formFieldGroups.push({
+        default: ['identityTokenAudience', 'identityTokenTtl'],
```

Comment:
should the key be `default` here? seems strange to have two default groups, I'd expect this to be
```suggestion
        wif: ['identityTokenAudience', 'identityTokenTtl'],
```

Reply:
Ah, this is because of how `formFieldGroups` works. "Default" means what shows above the toggle or what is not otherwise hidden behind a toggle. Long-term, we should consider a different pattern or maybe new component, but this was my work around for conditionally rendering fields based on the selection of another field. I remember Chelsea and I talking a bit about this logic though try as I might, I can't find the original thread.

<img width="528" alt="image" src="https://github.com/user-attachments/assets/3af3881f-a7c7-4bc6-bb77-0d7fc4d31cbf" />